api/DeviceAcceleration.json

```diff
@@ -5,10 +5,12 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/DeviceAcceleration",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": true,
+            "alternative_name": "DeviceMotionEventAcceleration"
```

Comment:
Can you file an issue about renaming this file instead? `DeviceMotionEventAcceleration` is the per-spec name after all.

Reply:
I'm assuming this would apply to `DeviceRotationRate` as well, correct?